cross/sonarr/Makefile

```diff
@@ -0,0 +1,35 @@
+PKG_NAME = Sonarr
+PKG_VERS = 4.0.0.344
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME).develop.$(PKG_VERS).linux-$(PKG_DIST_ARCH).$(PKG_EXT)
+PKG_DIST_SITE = https://download.sonarr.tv/v4/develop/$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)
+
+DEPENDS =
+
+UNSUPPORTED_ARCHS = $(DOTNET_UNSUPPORTED_ARCHS)
```

Comment:
Also, is this correct? Mono started to annoy me so I tried building v4 myself. Initially it gave an "unsupported arch" when I tried to compile it for armadaxp(armv7) DSM 7.1.

When I removed this it compiled fine and it also runs fine. So is the `DOTNET_UNSUPPORTED_ARCHS` still up to date?

Reply:
Yes, the `armadaxp` is not currently included as per:

https://github.com/SynoCommunity/spksrc/blob/175b8e87c5a170ceae5229994851ae4721511579/mk/spksrc.archs.mk#L53-L54

Reply:
Just FYI, Sonarr v4 does actually work on my DS414 (armadaxp). Sounds to me some applications do work and some don't.  😉

Reply:
Oh! I misinterpreted what you meant, thanks for clarifying. Perhaps you can take a look at the source issues https://github.com/SynoCommunity/spksrc/issues/4790 and https://github.com/SynoCommunity/spksrc/pull/4793. I don't have that model of NAS to do this myself but perhaps you can review the specific use cases and maybe re-test them to give a current position on this?

Looking across the codebase, this exclusion is currently made for jellyfin, lidarr, ombi, prowlarr as well as the dotnet-runtime. Ideally if you have something like Virtual DSM installed you can do some test deployments and gather current evidence to indicate the actual compatibility issues. With this evidence, a PR to include `armadaxp` in relevant .Net builds can be made (I'll help if you want).

Reply:
hey @lajuffermans, I've created a new issue for the testing of ARMv7 archs with ***arr** dotNET packages. I'd really appreciate your feedback to the issue to help with testing: https://github.com/SynoCommunity/spksrc/issues/5574